canonical/store-components
# Add logos to bundle cards

## Done
Use icons for charms in bundle cards

## QA
- Go to https://store-components-74.demos.haus/?path=/story/bundlecard--default-story
- Check that the charms in the bundle card are images (there won't be actual images because the URL has to be relative)
